mr-plonky2-circuits/src/storage/length_match.rs

```diff
@@ -184,6 +222,10 @@ impl Parameters {
         length_proof: &ProofWithPublicInputs<F, C, D>,
     ) -> Result<Vec<u8>> {
         let mut pw = PartialWitness::<F>::new();
+        // by default, we set to false, and then we change it for ERC20 query
+        // to accept the flag as argument.
```

Comment:
Don't we need to do it now in this PR?

Reply:
I want to merge this now as is on develop first then adapt the API to give false to this circuit. Didn't have the query erc api at that time.